core/src/test/java/org/apache/calcite/avatica/util/ArrayImplTest.java

```diff
@@ -164,6 +165,267 @@
     }
   }
 
+  /**
+   * The same test as arrayOfStructs(), except we use List instead of ArrayImpl.
+   *
+   * This test deliberately covers the
```

Comment:
Nitpick: I think the first line of the comment is enough, if you feel more context is needed, you can also refer to the JIRA ticket.